File: redwood-yoga-java/src/main/java/io/github/orioncraftmc/meditate/internal/enums/YGNodeType.java
package io.github.orioncraftmc.meditate.internal.enums;

public enum YGNodeType {
    YGNodeTypeDefault,
    YGNodeTypeText;

    public static final int SIZE = java.lang.Integer.SIZE;

    public static YGNodeType forValue(int value) {
        return values()[value];
    }

    public int getValue() {
        return this.ordinal();
    }
}

File: redwood-yoga/src/commonMain/kotlin/app/cash/redwood/yoga/internal/enums/YGNodeType.kt
package app.cash.redwood.yoga.internal.enums

enum class YGNodeType {
    YGNodeTypeDefault, YGNodeTypeText;

    fun getValue(): Int {
        return ordinal
    }

    companion object {
        fun forValue(value: Int): YGNodeType {
            return values()[value]
        }
    }
}
